### PR TITLE
List item checked

### PR DESCRIPTION
**Pull request by raffaelnagel**
Related to issue Allow tasks to be marked as completed / uncompleted #16

### CODE REVIEW

File: app/components/ListItem.js [hunk 1]
```diff
@@ -94,12 +117,27 @@ export default class ListItem extends React.Component {
...
+                id={this.state.id}
+              />
+              <label
+                htmlFor={this.state.id}
+              >
+                {' '}
+              </label>
+            </div>
+            <div className="col-md-9">
+              <span className={this.state.isChecked ? 'item-checked' : null} onClick={this.props.updateItem ? this.openEditMode : null}>
```
Comment (munetake) [inline app/components/ListItem.js]: Element properties can all go on their own line here.

File: app/components/ListItem.js [hunk 2]
```diff
@@ -63,6 +66,25 @@ export default class ListItem extends React.Component {
   }
 
   /**
+   * Makes check item call
+   */
+  handleCheck = () => {
+    this.setState({
```
Comment (munetake) [inline app/components/ListItem.js:72]: Should we still set the checkmark here? Even if an updateItem callback isn't passed?

File: app/styles/styles.css [hunk 3]
```diff
@@ -110,3 +119,108 @@ input:focus {
...
+}
+
+.list-item-component input[type='checkbox']:checked + label::after {
+  color: #ffffff;
+}
+
+.list-item-component .item-checked {
+  color: #888888;
+  text-decoration: line-through;
+}
```
Comment (munetake) [inline app/styles/styles.css]: Add newline at eof

File: app/components/ListItem.js [hunk 4]
```diff
@@ -94,12 +117,27 @@ export default class ListItem extends React.Component {
...
-              x
-            </button>
+            <div className="col-md-1">
+              <input
+                type="checkbox"
+                checked={this.state.isChecked}
+                onChange={this.handleCheck}
+                id={this.state.id}
+              />
+              <label
```
Comment (munetake) [inline app/components/ListItem.js]: This can go on a single line

File: app/styles/styles.css [hunk 5]
```diff
@@ -110,3 +119,108 @@ input:focus {
   cursor: pointer;
   background: transparent;
 }
+
+.list-item-component label {
```
Comment (munetake) [inline app/styles/styles.css:123]: If you could attach a couple of screenshots highlighting the new ui changes that would be awesome!

**Review by munetake — CHANGES_REQUESTED** [review]

**Comment by raffaelnagel** [comment]
All requested changes were committed.

**Pull request merged**